server/graphql/resolvers.ts

```diff
@@ -16,12 +16,19 @@ const resolvers = {
     requestGroups: (_, __, { dataSources }): Array<RequestGroupInterface> => dataSources.requestGroups.getAll()
   },
   RequestGroup: {
+    numOpen: (parent, __, { dataSources }): Array<RequestTypeInterface> => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)).requests.open.length).reduce((total, num) => total + num),
```

Comment:
Maybe use `numOpen` field on `RequestType`s instead of `request.open.length`

```
 numOpen: (parent, __, { dataSources }): Array<RequestTypeInterface> => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)).numOpen).reduce((total, num) => total + num)
```

Reply:
The parent returns objects, not a part of the graphql query, so there would be no actual numOpen property on these objects

Reply:
icic